deeplearning4j-keras/src/main/java/org/deeplearning4j/keras/DeepLearning4jEntryPoint.java

```diff
@@ -0,0 +1,112 @@
+package org.deeplearning4j.keras;
+
+import lombok.extern.slf4j.Slf4j;
+import org.bytedeco.javacpp.FloatPointer;
+import org.bytedeco.javacpp.hdf5;
+import org.deeplearning4j.nn.modelimport.keras.InvalidKerasConfigurationException;
+import org.deeplearning4j.nn.modelimport.keras.KerasModelImport;
+import org.deeplearning4j.nn.modelimport.keras.UnsupportedKerasConfigurationException;
+import org.deeplearning4j.nn.multilayer.MultiLayerNetwork;
+import org.nd4j.linalg.api.ndarray.INDArray;
+import org.nd4j.linalg.factory.Nd4j;
+import org.nd4j.linalg.indexing.INDArrayIndex;
+import org.nd4j.linalg.indexing.NDArrayIndex;
+import org.nd4j.linalg.util.ArrayUtil;
+
+import java.io.IOException;
+
+import static org.bytedeco.javacpp.hdf5.H5F_ACC_RDONLY;
+
+@Slf4j
+public class DeepLearning4jEntryPoint {
+
+    public void fit(EntryPointFitParameters entryPointFitParameters)
+            throws IOException, InterruptedException, InvalidKerasConfigurationException, UnsupportedKerasConfigurationException {
+
+        try {
+            MultiLayerNetwork multiLayerNetwork;
+            if ("sequential".equals(entryPointFitParameters.getType())) {
+                multiLayerNetwork = KerasModelImport.importKerasSequentialModelAndWeights(entryPointFitParameters.getModelFilePath());
+                multiLayerNetwork.init();
+            } else {
+                throw new RuntimeException("Model type unsupported! (" + entryPointFitParameters.getType() + ")");
+            }
+
+            INDArray features = h5FileToNDArray(entryPointFitParameters.getTrainFeaturesFile());
+            INDArray labels = h5FileToNDArray(entryPointFitParameters.getTrainLabelsFile());
+
+            INDArrayIndex[] ndIndexes = new INDArrayIndex[features.shape().length];
+            for (int i = 0; i < ndIndexes.length; i++) {
+                ndIndexes[i] = NDArrayIndex.all();
+            }
+
+            for (int i = 0; i < entryPointFitParameters.getNbEpoch(); i++) {
+                log.info("Fitting: " + i);
```

Comment:
This is a lot of code. Could we maybe have a unit test for *just* this section? I would encapsulate this in a method. The views here are my main concern.